Errors after upgrading app from framework 6 to 7.

### Description

After update to Visual Studio 2022 version 17.4.0
and
after upgrading a NET.MAUI app from framework 6 to framework 7 by changing the target .NET Runtime from net6.0 to net 7.0 in the project file I get four times this error:

NU1105 Unable to read project information for 'Finance': Sequence contains more than one element.

After that I see that all the installed NuGet Packages are gone.
I tried to reinstall them but I get the error:
Invalid restore input.
Missing required property 'ProjectName'.  Input files: C:\Sources\MAUI\Finance\Finance\Finance.csproj.

Missing NuGet packages:
Microsoft.Extensions.Localization
Microsoft.Maui.Dependencies
Microsoft.Maui.Extensions
Microsoft.Windows.SDK.BuildTools
System.Runtime.InteropServices.NFloat.Internal

Any solution for this?
Thanks in advance.

### Steps to Reproduce

1. upgrading a NET.MAUI app from framework 6 to framework 7 by changing the target .NET Runtime from net6.0 to net 7.0 in the project file.

### Link to public reproduction project repository

https://github.com/geertgeerits/Finance

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Pixel 5 API 31 Android 12.0

### Did you find any workaround?

No

### Relevant log output

_No response_

Im stuck with below errors after upgrade, tried leave only windows target, currently left with android and windows. It is weird because this lib not using markup, community toolkit only referenced because of IMessenger.
```
1>...\CommunityToolkit.Maui.Markup.SourceGenerators\CommunityToolkit.Maui.Markup.SourceGenerators.TextAlignmentExtensionsGenerator\EditorTextAlignmentExtensions.g.cs(122,147,122,167): error CS0121: The call is ambiguous between the following methods or properties: 'CommunityToolkit.Maui.Markup.TextAlignmentExtensions_Editor.TextCenterHorizontal(Microsoft.Maui.Controls.Editor)' and 'CommunityToolkit.Maui.Markup.TextAlignmentExtensions_Editor.TextCenterHorizontal(Microsoft.Maui.Controls.Editor)'
1>...\CommunityToolkit.Maui.Markup.SourceGenerators\CommunityToolkit.Maui.Markup.SourceGenerators.TextAlignmentExtensionsGenerator\EntryTextAlignmentExtensions.g.cs(122,145,122,165): error CS0121: The call is ambiguous between the following methods or properties: 'CommunityToolkit.Maui.Markup.TextAlignmentExtensions_Entry.TextCenterHorizontal(Microsoft.Maui.Controls.Entry)' and 'CommunityToolkit.Maui.Markup.TextAlignmentExtensions_Entry.TextCenterHorizontal(Microsoft.Maui.Controls.Entry)'
1>...\CommunityToolkit.Maui.Markup.SourceGenerators\CommunityToolkit.Maui.Markup.SourceGenerators.TextAlignmentExtensionsGenerator\LabelTextAlignmentExtensions.g.cs(122,145,122,165): error CS0121: The call is ambiguous between the following methods or properties: 'CommunityToolkit.Maui.Markup.TextAlignmentExtensions_Label.TextCenterHorizontal(Microsoft.Maui.Controls.Label)' and 'CommunityToolkit.Maui.Markup.TextAlignmentExtensions_Label.TextCenterHorizontal(Microsoft.Maui.Controls.Label)'
1>...\CommunityToolkit.Maui.Markup.SourceGenerators\CommunityToolkit.Maui.Markup.SourceGenerators.TextAlignmentExtensionsGenerator\PickerTextAlignmentExtensions.g.cs(122,147,122,167): error CS0121: The call is ambiguous between the following methods or properties: 'CommunityToolkit.Maui.Markup.TextAlignmentExtensions_Picker.TextCenterHorizontal(Microsoft.Maui.Controls.Picker)' and 'CommunityToolkit.Maui.Markup.TextAlignmentExtensions_Picker.TextCenterHorizontal(Microsoft.Maui.Controls.Picker)'
```

--- a/src/Controls/src/Core/Entry.cs
+++ b/src/Controls/src/Core/Entry.cs
@@ -94,9 +94,9 @@ namespace Microsoft.Maui.Controls
 		public static readonly BindableProperty SelectionLengthProperty = BindableProperty.Create(nameof(SelectionLength), typeof(int), typeof(Entry), 0, validateValue: (b, v) => (int)v >= 0);
 
 		/// <summary>
-		/// Backing store for the <see cref="SelectionLength"/> property.
+		/// Backing store for the <see cref="ClearButtonVisibility"/> property.
 		/// </summary>
-		public static readonly BindableProperty ClearButtonVisibilityProperty = BindableProperty.Create(nameof(SelectionLength), typeof(ClearButtonVisibility), typeof(Entry), ClearButtonVisibility.Never);
+		public static readonly BindableProperty ClearButtonVisibilityProperty = BindableProperty.Create(nameof(ClearButtonVisibility), typeof(ClearButtonVisibility), typeof(Entry), ClearButtonVisibility.Never);
 
 		readonly Lazy<PlatformConfigurationRegistry<Entry>> _platformConfigurationRegistry;
 

